t2.js

```diff
@@ -0,0 +1,5 @@
+//
+if (!!t2)
```

Comment:
Codacy has a fix for the issue: [Redundant double negation.](https://app.codacy.com/gh/ignatios-varelas/test-repo/pullRequest?prid=7374823)
```suggestion
if (t2)
```